CLAUDE.md

```diff
@@ -0,0 +1,30 @@
+# Bitwarden Internal SDK
+
+Rust SDK centralizing business logic **Internal use only**
+
+## Client Pattern
+
+PasswordManagerClient ([bitwarden-pm](crates/bitwarden-pm/src/lib.rs)) wraps
+[bitwarden_core::Client](crates/bitwarden-core/src/client/client.rs) and exposes sub-clients:
+`auth()`, `vault()`, `crypto()`, `sends()`, `generator()`, `exporters()`.
+
+**Lifecycle**: Init → Lock/Unlock → Logout (drops instance). Memento pattern for state resurrection.
+
+**Storage**: Consuming apps use `HashMap<UserId, PasswordManagerClient>`.
+
+## Crate Organization
+
+- `bitwarden-core` - Core Client struct (avoid editing, use feature crates)
+- `bitwarden-crypto` - Crypto primitives (edit with extreme care, multi-team ownership)
+  - `derive_*` = deterministic key derivation, `make_*` = non-deterministic generation
+  - Memory zeroed on drop by default
+- `bitwarden-{auth,vault,send,generators}` - Domain features
+- `bitwarden-uniffi` - Mobile bindings (no lifetimes in FFI types)
+- `bitwarden-wasm-internal` - Web bindings (no logic, only conversions)
+- `bitwarden-api-*` - Auto-generated (regenerate via `./support/build-api.sh`, revert Cargo.toml)
+
+## Non-Obvious Constraints
+
+- Clippy allows `.unwrap()` and `.expect()` in tests only
```

Comment:
I believe the expect part is not correct. We allow expect, but it should obviously be used with care: https://github.com/bitwarden/sdk-internal/blob/ae9b8b5241b00871b31efecb1de4c60f10a8d9fd/crates/bitwarden-crypto/src/rsa.rs#L28